internal/http_api/api_response.go

```diff
@@ -43,11 +42,9 @@ func PlainText(f APIHandler) APIHandler {
 		}
 		switch d := data.(type) {
 		case string:
-			w.Header().Set("Content-Length", strconv.Itoa(len(d)))
```

Comment:
How is it possible for it to be able to calculate the length to provide this automatically for us _before_ we've written the body - it doesn't know the size of the body to expect?